### PR TITLE
Spaces index page

### PR DESCRIPTION
**Pull request by TomLouwagie**
Here my pull request for spaces#index and the corresponding view. I also added some colors and components.

### CODE REVIEW

File: app/assets/stylesheets/config/_colors.scss [hunk 1]
```diff
@@ -1,10 +1,15 @@
-// Define variables for your color scheme
+// Custom custudyo colors
```
Comment (pkonstantinou) [inline app/assets/stylesheets/config/_colors.scss:1]: typo: custudyo -> costudyo

File: app/assets/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.card {
+  height: 360px;
+  background-color: $light-pink;
+  color: $black;
+}
+
+.btn:hover {
+  text-decoration: none !important;
+  // background-color: red;
```
Comment (pkonstantinou) [inline app/assets/stylesheets/components/_card.scss:9]: Forgotten comment

File: app/assets/stylesheets/components/_card.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.card {
+  height: 360px;
+  background-color: $light-pink;
+  color: $black;
+}
+
+.btn:hover {
+  text-decoration: none !important;
```
Comment (pkonstantinou) [inline app/assets/stylesheets/components/_card.scss:8]: Why are you exactly overwriting this? And why in the _card.scss file?

File: app/assets/stylesheets/components/_btn.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.btn-ghost:hover {
+  opacity: 1;
+}
+
+.btn-flat {
+  padding: 8px 24px;
+  border-radius: 4px;
+  background: $custodyo-gray;
```
Comment (pkonstantinou) [inline app/assets/stylesheets/components/_btn.scss:18]: typo: $custodyo-gray -> $costudyo-gray

**Review by pkonstantinou — CHANGES_REQUESTED** [review]
Nothing important, you just had few typos basically.

File: app/assets/stylesheets/components/_card.scss (hunk 3 above)
Comment (TomLouwagie) [reply app/assets/stylesheets/components/_card.scss:8]: Your right, I was experimenting with removing the "link" styling, but we don't need it. I took it out.

File: app/assets/stylesheets/components/_btn.scss (hunk 4 above)
Comment (TomLouwagie) [reply app/assets/stylesheets/components/_btn.scss:18]: changed it

**Pull request merged**